src/NorthwindWebsite/Configuration/NginxConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.AspNetCore.HttpOverrides;
+
+namespace NorthwindWebsite.Configuration;
+
+public static class NginxConfiguration
+{
+    public static void ConfigureNginxServer(this WebApplication app) =>
```

Comment:
Just for information: `Nginx` here is not really suitable since it makes your config specific to particular server. However, if you decide to switch to Apache, as an example, you would probably have to re-write this config. So, let's name it just as `ConfigureProxyServer`